src/DemuxTopic.h

```diff
@@ -3,6 +3,7 @@
 #include "Source.h"
 #include "TimeDifferenceFromMessage.h"
 #include "json.h"
+#include <chrono>
```

Comment:
Was this intentional?

Reply:
Yes, this was intentional. I have not looked at the "include paths" but when I removed Alloc.h, some of the build nodes complained about missing std::chronological. With that said, you do have a point in that those include-statements could (should) probably be in the cpp-file.